### ISSUE TITLE
Nonexistent reference-style links don't cause an error

### ISSUE THREAD

**Issue by mieubrisse** [remarkjs/remark-validate-links#74]
### Initial checklist

- [X] I read the [support docs](https://github.com/remarkjs/.github/blob/main/support.md)
- [X] I read the [contributing guide](https://github.com/remarkjs/.github/blob/main/contributing.md)
- [X] I agree to follow the [code of conduct](https://github.com/remarkjs/.github/blob/main/code-of-conduct.md)
- [X] I searched [issues](https://github.com/search?q=user%3Aremarkjs&type=Issues) and couldn’t find anything (or linked relevant results below)

### Affected packages and versions

12.1.0

### Steps to reproduce

Run on this Markdown:

```markdown
[This link doesn't go anywhere][this-is-nonexistent]
```

### Expected behavior

The link-validating should throw an error due to `this-is-nonexistent` reference not existing.

### Actual behavior

Validation passes without issue

### Runtime

Node v17

### Package manager

yarn 1

### OS

macOS

### Build and bundle tools

Docusaurus

**Comment by remcohaszing** [comment]
You can use [`remark-lint-no-undefined-references`](https://github.com/remarkjs/remark-lint/tree/main/packages/remark-lint-no-undefined-references) for that.

**Comment by mieubrisse** [comment]
Ah great, thank you!
